cpp/include/cuvs/core/c_api.h

```diff
@@ -48,6 +49,35 @@ void cuvsSetLastErrorText(const char* error);
 
 /** @} */
 
+/**
+ * @defgroup log_c cuVS Logging
```

Comment:
Can you add an entry for this to the C docs in `docs/source/c_api/core_c_api.rst` ? (following the same pattern as the error handling / `error_c` section should be sufficient)